packages/react-openapi/src/OpenAPISchema.tsx

```diff
@@ -172,12 +172,7 @@ export function OpenAPIRootSchemaFromServer(props: {
     schema: string;
     context: OpenAPIClientContext;
 }) {
-    return (
-        <OpenAPIRootSchema
-            schema={JSON.parse(props.schema, retrocycle())}
-            context={props.context}
-        />
-    );
+    return <OpenAPIRootSchema schema={safeJSONParse(props.schema)} context={props.context} />;
```

Comment:
I'm missing context here, does it mean we parse the entire props passed from a server component?

It must be huge in the HTML, why are we doing this?